app/assets/stylesheets/new_board/_screen.scss

```diff
@@ -1,3 +1,41 @@
 .full-heigth {
     height:100%;
 }
+
+.markdown-wrapper{
```

Comment:
You've forgotten a space here: `.markdown-wrapper {`. There are other parts with the same mistake.